src/WebSdk/Worker/Targets/Microsoft.NET.Sdk.Worker.targets

```diff
@@ -25,4 +26,15 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ProjectCapability Include="WebNestingDefaults" />
   </ItemGroup>
 
+
+  <ItemGroup Condition=" '$(DisableImplicitNamespaceImports_Worker)' != 'true'
+      And '$(TargetFrameworkIdentifier)' == '.NETCoreApp'
+      And $([MSBuild]::VersionGreaterThanOrEquals('$(_TargetFrameworkVersionWithoutV)', '6.0'))">
+    <Import Include="System.Threading" />
```

Comment:
Should this be added to the base .NET SDK instead of here?

Reply:
+1